cli_helpers/_compat.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+"""Python 2/3 compatibility support."""
```

Comment:
This compat module is for internal utils, right? Why not move it into `__init__.py`, so you would not have to import things from an underscored package name?